pkgconfig/README.md

```diff
@@ -52,8 +52,9 @@ $ clang++-11 ${cxxflags} -o enc enc.cpp ${libs}
 To build an enclave application with a specific crypto library, additional commands are needed to incorporate additional source files and headers. In this example, `OE_CRYPTO_LIB` is set to `openssl`. The values that `OE_CRYPTO_LIB` supports are:
 - `mbedtls`
 - `openssl`
-- `openssl_symcrypt_fips`
+- `symcrypt_fips`
 - `openssl_3`
+- `symcrypt_prov_fips`
```

Comment:
can we do
`openssl_3_symcrypt_fips`
for consistency?

Reply:
In general, we want `OE_CRYPTO_LIB` to indicate what crypto library the sample uses. In this case, we need to say that it's openssl 3 backed by SymCrypt FIPS module. If we want to emphasis on the use of provider, then it can be `openssl_3_symcrypt_prov_fips`.

Reply:
I made this change